raiden-cli/src/index.ts

```diff
@@ -73,11 +73,19 @@ async function raidenEnsureChannelIsOpen(
   return channel;
 }
 
-async function raidenTransfer(token: string, target: string, value: BigNumberish) {
+async function raidenTransfer(
+  token: string,
+  target: string,
+  opts: { amount: BigNumberish; identifier?: BigNumberish },
```

Comment:
Please move `amount` out of `opts`. It doesn't make sense that `amount` is an option and `token` and `target isnt`

Reply:
It does, per API: token and target are URL params on Raiden API, while amount and identifier comes as body params. This is perfectly mirrored here. But I don't have hard feelings about it, and will extract.

Reply:
Done

Reply:
> token and target are URL params on Raiden API, while amount and identifier comes as body params

Apples and oranges. One is how the end user uses it, the other is internal implementation